src/Sulu/Bundle/AdminBundle/Resources/js/views/FormOverlayList/formOverlayList.scss

```diff
@@ -0,0 +1,7 @@
+@import '../../components/Overlay/variables.scss';
+@import '../../components/ColumnList/variables.scss';
+
+.form {
+    padding: 30px;
```

Comment:
In all other overlays we have used the `$viewPadding` variable for that, haven't we?